resources/devices/mobile-phone/samsung.json

```diff
@@ -5628,6 +5628,19 @@
             "isTablet": "false"
         }
     },
+    "Samsung SM-G530BT": {
```

Comment:
This one and the one below aren't used in any changes in this diff, but some other changes coming as part of #1344. Since I had made some other changes to this file, I just added those in now.